.snippets/code/tutorials/interoperability/uniswapv2-swap/terminal/send-xcm.md

```diff
@@ -0,0 +1,3 @@
+<div id="termynal" data-termynal>
```

Comment:
since this doesn't show any output, it'd  be better to embed this into the page and wrap it in triple back ticks instead so at least the user will be able to easily copy the command